### DISCUSSION TITLE
saf-training/courses/guidance/02

### DISCUSSION THREAD

**Discussion by giscus** [mitre/saf-training#269] [Ideas]
# saf-training/courses/guidance/02

2.1 Security Guidance Before we go further, let's discuss what we mean by "security guidance" and some of its characteristics that matter to us as security (and automation!) pra...

https://mitre.github.io/saf-training/courses/guidance/02.html

**Comment by karikarshivani** [comment]
2.1.1 security*

**Comment by karikarshivani** [comment]
A resource recommendation by a student: https://ncp.nist.gov/repository

**Comment by p-oneil** [comment]
Moved to PR
